modules/aws/dynamodb.go

```diff
@@ -0,0 +1,80 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/stretchr/testify/require"
+)
+
+// GetDynamoDbTableTags fetches resource tags of a specified dynamoDB table.
+func GetDynamoDbTableTags(t *testing.T, region string, tableName string) []*dynamodb.Tag {
+	tags, err := GetDynamoDbTableTagsE(t, region, tableName)
+	require.NoError(t, err)
+	return tags
+}
+
+// GetDynamoDbTableTagsE fetches resource tags of a specified dynamoDB table.
+func GetDynamoDbTableTagsE(t *testing.T, region string, tableName string) ([]*dynamodb.Tag, error) {
+	table := GetDynamoDBTable(t, region, tableName)
+	out, err := NewDynamoDBClient(t, region).ListTagsOfResource(&dynamodb.ListTagsOfResourceInput{
+		ResourceArn: table.TableArn,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return out.Tags, err
+}
+
+// GetDynamoDBTableTimeToLive fetches information about TTL configuration of a specified dynamoDB table.
```

Comment:
```suggestion
// GetDynamoDBTableTimeToLive fetches information about the TTL configuration of a specified dynamoDB table. This will fail the test if there are any errors.
```